docs/BR.md

```diff
@@ -1321,9 +1328,7 @@ defined by RFC6960.
 
 ### 4.9.10 On-line revocation checking requirements
 
-The CA MAY support on-line revocation status checking using OCSP.
-
-If the CA supports OCSP, the following requirements SHALL apply:
+If the CA signs OCSP responses either directly or through a delegated OCSP responder, the following requirements SHALL apply.
```

Comment:
I do like this phrasing update. However, it doesn't fully answer my question:

Is it acceptable for a CA to operate an OCSP service for only *some* of the non-expired certificates it has issued?

For example, a CA might switch from including an AIA OCSP url in its certificates, to including a CRLDP url in its certificates, with no overlap: each Subscriber cert contains only one or the other. Is the CA required to provide OCSP responses, and update those responses no later than four days after the thisUpdate, even for the certificates that do not embed an AIA OCSP url?

The previous phrasing ("support on-line revocation status checking using OCSP") and the current phrasing ("signs OCSP responses") both seem to assume that OCSP support is a binary thing: either the CA does it, or it doesn't. This makes sense both in today's world when OCSP is required, and in the potential future when OCSP is wholly optional. It raises questions about the transition period, however.

To be clear, I don't think this is insurmountable. For example, a CA could choose to stop including AIA OCSP urls, but continue providing OCSP responses for all certificates until all certificates with an AIA OCSP url have expired, then instantaneously stop providing any OCSP responses at all. But since many CAs will be going through a transition period here, some clarity if possible would be nice.

Reply:
In the CP for another PKI that is based on the BRs and EVGs, we made OCSP optional for Subscriber Certificates. To make it abundantly clear that the OCSP requirements were relevant only to those certificates that the CA "intended" to support OCSP for, we used this turn of phrase:

```
For the status of Subscriber Certificates which include an Authority Information Access extension with a id-
ad-ocsp accessMethod (“AIA OCSP pointer”)
```

We may want to do similar here.

Reply:
LGTM

Reply:
I like this approach